### ISSUE TITLE
Dimmer - does not accept simple brightness change

### ISSUE THREAD

**Issue by Paul-Reed** [windkh/node-red-contrib-shelly#19]
The dimmer can be switched on, or off by the command;
```
{
    on : true  //or on : false
}
```
but the brightness cannot be varied by;
```
{
    brightness: 26
}
```
and instead needs the full object to change the brightness;
```
{
    on : true,
    brightness: 100
}
```
which seems a bit inconsistent.

My user case is that the flow will not know if the light is switched on or off, and it's only job is to set the brightness, but I cannot do that at the moment, as I would need to include `on : true` in the object.
(That may sound pointless if the light is currently switched off, but the brightness would remain preset at that level until it was switched on either via voice command or hardwired switch, and then would assume the preset brightness).

I'm currently using MQTT, and can send `{ brightness : 66 }` which works OK, so this should be possible.

Many thanks.

**Comment by Paul-Reed** [comment]
Pull request - https://github.com/windkh/node-red-contrib-shelly/pull/20 created to add feature

**Comment by windkh** [comment]
merged thanks: see version 1.6.0
